src/index.js

```diff
@@ -40,6 +40,57 @@ const _views = new WeakMap();
 
 let bypass = false;
 
+function nextOrPreviousProject (index) {
+    // Generate an array of projects by selecting children of 'project-viewer > ul[is="pv-list-tree"]'. For each 'li[is="pv-list-nested-item"]', expand it into its children.
+    const sidebar = document.getElementsByTagName('project-viewer')[0];
+    let list = sidebar.getElementsByTagName('ul')[0].children;
+
+    var output = [];
+
+    var expandProjects = function (element) {
+        if (element.getAttribute('is') === "pv-list-nested-item") {
+            let nestedList = element.getElementsByTagName('ul')[0].children;
```

Comment:
> So here is where i have problems because my `element.getElementsByTagName('ul')[0]` is an empty `ul` and I actually have more elements.

That's so weird. So the DOM is displaying differently in your Atom than mine? Mind sharing the compiled source from the dev tools so that I can better understand what's going on? Here's what I see:
http://puu.sh/q5b6x/0cc51573d8.png

`project-viewer` always has exactly two children, a `pv-header` and a `ul[is="pv-list-tree"]`. If that lone `ul` is blank, then that would mean that there *are* no projects in the sidebar and the command should quit because there's nothing to do.

I think I've found that bug, though. It happens to me when there's a client in the list (I hadn't been using them), regardless of whether there's anything under it, and the active project is at `output[0]`. When the active project is anywhere else on the list, the navigation works fine until it reaches the beginning. Can you confirm?